src/components/LocationForm.js

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from "react";
```

Comment:
Prettier was not applied on this file. You can install an extension that's called prettier and then there should be an option to apply it on save in the settings (which would mean that prettier gets applied every time you save something).